deploy/gke/dev.yaml

```diff
@@ -21,3 +21,7 @@ region:
     - europe-west2
 nodes: 1
 storage_project: datcom-store
+# Network name used for VPC peerings.
+network_name: "default"
+# Assumes a directory with this name exists in gs://datcom-website-dev-modelstore/
```

Comment:
I guess you also want a field for the model bucket?

Reply:
Done.